### PR TITLE
[zh]sync-setup-3

### PR DESCRIPTION
**Pull request by tengqm**
This one supersedes #33625.
Co-Authored-By: @wlp1153468871 
related:  #33475

### CODE REVIEW

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -24,32 +26,131 @@ what is involved and describes related tasks for setting up nodes.
...
-For other operating systems, look for documentation specific to your platform.
+Kubernetes releases before v1.24 included a direct integration with Docker Engine, using a component named _dockershim_. That special direct integration is no longer part of Kubernetes (this removal was [announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation) as part of the v1.20 release). You can read [Check whether Dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) to understand how this removal might affect you. To learn about migrating from using dockershim, see [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
+
+If you are running a version of Kubernetes other than v{{< skew currentVersion >}}, check the documentation for that version.
 {{< /note >}}
+
+ -->
+ {{< note >}}
+提示：v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。 
+这种特殊的直接整合不再是Kubernetes 的一部分
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:57]:
```suggestion
这种特殊的直接整合不再是 Kubernetes 的一部分
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 2]
```diff
@@ -58,7 +159,7 @@ Systemd has a tight integration with cgroups and allocates a cgroup per systemd
 to configure your container runtime and the kubelet to use `cgroupfs`. Using `cgroupfs` alongside
 systemd means that there will be two different cgroup managers.
 -->
-控制组用来约束分配给进程的资源。
+在 Linux 上，{{<glossary_tooltip text="控制组（CGroup）" term_id="cgroup" >}}用于限制分配给进程的资源。
 
 当某个 Linux 系统发行版使用 [systemd](https://www.freedesktop.org/wiki/Software/systemd/)
 作为其初始化系统时，初始化进程会生成并使用一个 root 控制组 (`cgroup`), 并充当 cgroup 管理器。
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:165]:
```suggestion
作为其初始化系统时，初始化进程会生成并使用一个 root 控制组（`cgroup`），并充当 cgroup 管理器。
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 3]
```diff
@@ -107,12 +208,12 @@ configuration, or reinstall it using automation.
...
 
 <!--
-## Cgroup v2
+### Cgroup version 2 {#cgroup-v2}
 
 Cgroup v2 is the next version of the cgroup Linux API.  Differently than cgroup v1, there is a single
 hierarchy instead of a different one for each controller.
 -->
-## Cgroup v2
+### Cgroup 版本 2 {#cgroup-v2}
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:216]:
```suggestion
### Cgroup 第 2 版 {#cgroup-v2}
```

**Review by Sea-n — COMMENTED** [review]
LGTM, just a few nitpicks.

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 4]
```diff
@@ -24,32 +26,131 @@ what is involved and describes related tasks for setting up nodes.
...
+
+If you are running a version of Kubernetes other than v{{< skew currentVersion >}}, check the documentation for that version.
 {{< /note >}}
+
+ -->
+ {{< note >}}
+提示：v1.24 之前的 Kubernetes 版本包括与 Docker Engine 的直接集成，使用名为 _dockershim_ 的组件。 
+这种特殊的直接整合不再是Kubernetes 的一部分
+（这次删除是[宣布](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)作为 v1.20 版本的一部分）。
+你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/)
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:59]: Broken link.
```suggestion
你可以阅读[检查 Dockershim 弃用是否会影响你](/zh/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) 
```

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 5]
```diff
@@ -24,32 +26,131 @@ what is involved and describes related tasks for setting up nodes.
...
 - [containerd](#containerd)
 - [CRI-O](#cri-o)
-- [Docker](#docker)
+- [Docker Engine](#docker)
+- [Mirantis Container Runtime](#mcr)
 
 <!-- 
 {{< note >}}
-For other operating systems, look for documentation specific to your platform.
+Kubernetes releases before v1.24 included a direct integration with Docker Engine, using a component named _dockershim_. That special direct integration is no longer part of Kubernetes (this removal was [announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation) as part of the v1.20 release). You can read [Check whether Dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim check-if-dockershim-deprecation-affects-you/) to understand how this removal might affect you. To learn about migrating from using dockershim, see [Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:49]: The link break was modified by the original translator.
In `en` content, the link was fine.

```suggestion
Kubernetes releases before v1.24 included a direct integration with Docker Engine,
using a component named _dockershim_. That special direct integration is no longer
part of Kubernetes (this removal was
[announced](/blog/2020/12/08/kubernetes-1-20-release-announcement/#dockershim-deprecation)
as part of the v1.20 release).
You can read
[Check whether Dockershim deprecation affects you](/docs/tasks/administer-cluster/migrating-from-dockershim/check-if-dockershim-deprecation-affects-you/) to understand how this removal might
affect you. To learn about migrating from using dockershim, see
[Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/).
```

**Review by Sea-n — COMMENTED** [review]
Netlify preview: <https://deploy-preview-33981--kubernetes-io-main-staging.netlify.app/zh/docs/setup/production-environment/container-runtimes/>

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 6]
```diff
@@ -58,10 +169,10 @@ Systemd has a tight integration with cgroups and allocates a cgroup per systemd
 to configure your container runtime and the kubelet to use `cgroupfs`. Using `cgroupfs` alongside
 systemd means that there will be two different cgroup managers.
 -->
-控制组用来约束分配给进程的资源。
+在 Linux 上，{{<glossary_tooltip text="控制组（CGroup）" term_id="cgroup" >}}用于限制分配给进程的资源。
 
 当某个 Linux 系统发行版使用 [systemd](https://www.freedesktop.org/wiki/Software/systemd/)
-作为其初始化系统时，初始化进程会生成并使用一个 root 控制组 (`cgroup`), 并充当 cgroup 管理器。
+作为其初始化系统时，初始化进程会生成并使用一个 root 控制组（`cgroup`）, 并充当 cgroup 管理器。
```
Comment (Sea-n) [inline content/zh/docs/setup/production-environment/container-runtimes.md:175]:
```suggestion
作为其初始化系统时，初始化进程会生成并使用一个 root 控制组（`cgroup`），并充当 cgroup 管理器。
```

**Review by Sea-n — COMMENTED** [review]
Overall LGTM

I think other fixes can leave it to next contributor. 😄

**Comment by mengjiao-liu** [comment]
/approve
/lgtm
Thanks!

**Pull request merged**